main.yml

```diff
@@ -171,7 +185,8 @@ migration-analytics:
   disabled_on_stable: true
   frontend:
     paths:
-      - /staging/migration-analytics
```

Comment:
If you're removing it from `/staging`, you should also remove the reference from the Staging bundle.

Reply:
Removed reference from staging sub_apps